services/portal/exportToWorkspace/exportToWorkspaceSequences.js

```diff
@@ -1,56 +1,44 @@
 const exportToWorkspaceQuestions = require('./exportToWorkspaceQuestions.js');
 const exportToWorkspaceTasks = require('./exportToWorkspaceTasks.js');
-const exportToWorkspaceProps = require('./exportToWorkspaceProps.js');
 
 const I = actor();
 
 /**
  * exportToWorkspace Sequences
  */
 module.exports = {
-  /* The 'Export default manifest, mount it and check manifest name' test sequence, steps are:
-    - from 'Explore' page, click 'Export to workspace' button to export default manifest to workspace
-    - validate the message toaster and grab the filename of exported manifest file
-    - redirect to workspace and spawn a workspace if none exists
-    - create a python notebook in workspace and mount the latest manifest using python code
-    - verify the mounted file has the same filename as the exported manifest file
-    - go back to workspace and delete all Jupyther notebooks for clean up */
-  async checkExportDefaultManifestToWorkspace() {
+  /* The 'Export default manifest, mount it and check manifest filename' test sequence */
+  async checkExportDefaultManifestToWorkspace(exportToWorkspaceUtil) {
     exportToWorkspaceTasks.exportDefaultManifestToWorkspace();
     exportToWorkspaceQuestions.isManifestSavedToWorkspaceSucceeded();
     const manifestFilename = await exportToWorkspaceTasks.grabManifestFilename();
     exportToWorkspaceTasks.jumpToWorkspacePage();
     exportToWorkspaceTasks.startWorkspace();
-    const mountOutput = await exportToWorkspaceTasks.mountLatestManifestInJupyterNotebook();
+    const mountOutput = await exportToWorkspaceTasks.mountLatestManifestInJupyterNotebook(exportToWorkspaceUtil);
     await exportToWorkspaceQuestions.doesMountOutputLookSuccessful(mountOutput, manifestFilename);
-    await exportToWorkspaceTasks.backToWorkspace();
+    await exportToWorkspaceTasks.backToWorkspace(exportToWorkspaceUtil);
```

Comment:
`mountLatestManifestInJupyterNotebook` and `backToWorkspace` don't take a parameter